template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/CmDBConfigProviderTest.java

```diff
@@ -0,0 +1,117 @@
+package com.sequenceiq.cloudbreak.cmtemplate;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
+import com.cloudera.api.swagger.model.ApiClusterTemplateService;
+import com.sequenceiq.cloudbreak.util.FileReaderUtils;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CmDBConfigProviderTest {
+
+    private CmTemplateProcessor underTest;
+
+    @Test
+    public void testSqoopServiceDBConfigs() {
```

Comment:
Can you please separate this file and create one for envery service type ?

Reply:
Sure, separated the test files into once for each service. Thanks